ui/bits/css/user/_show.scss

```diff
@@ -93,9 +93,10 @@
       }
       &:hover {
         > a {
-          @extend %box-radius-top, %dropdown-shadow;
+          @extend %box-radius-top;
           background: $c-bg-header-dropdown;
           color: $c-font-clear;
+          box-shadow: -1px 5px 6px rgba(0, 0, 0, 0.3);
```

Comment:
looks like a solution for the transparent theme, but what does it look like in light theme?

Reply:
It looks the same as it did before, it's the same [box-shadow](https://github.com/lichess-org/lila/blob/67057907e67974841044ce9c498d942a683478d5/ui/common/css/abstract/_extends.scss#L110) that `dropdown-box` was using.

![image](https://github.com/lichess-org/lila/assets/3620552/7de47dd2-666a-4a85-b58e-9fe2f3e6d7f0)

![image](https://github.com/lichess-org/lila/assets/3620552/6909c4e3-2e17-4857-a078-6a36355f61a7)

![image](https://github.com/lichess-org/lila/assets/3620552/44e9644b-080b-4cdf-bdcc-ab61ec10db58)